tests/spread/integration/busybox-static/task.yaml

```diff
@@ -0,0 +1,7 @@
+summary: Integration tests for busybox-static
+
+execute: |
+  rootfs="$(install-slices busybox-static_bins)"
+
+  test $(chroot "${rootfs}" busybox echo "Success") == "Success"
+  test $(chroot "${rootfs}" static-sh -c 'echo ahoy') == "ahoy"
```

Comment:
FYI, I believe the correct syntax for testing string equality in a shell is `=`, not `==`.

Reply:
```shell
$ man test
TEST(1)                                                                                                                                              User Commands                                                                                                                                             TEST(1)

NAME
       test - check file types and compare values
...
       STRING1 = STRING2
              the strings are equal
...
```